src/theme/protocolTheme.ts

```diff
@@ -23,6 +23,10 @@ const lightMode = {
     secondary: Colors.SecondaryText,
     disabled: Colors.TextDisabled,
   },
+  icons: {
+    success: Colors.IconLightSuccess,
+    error: Colors.IconLightError,
+  },
```

Comment:
I would have added all of them to `text` above but I don't mind either, we can always revisit if we see it's useful for text as well.

Reply:
I would go with revisit later for this one. I don't see it used on texts yet.